doc/releases/changelog-dev.md

```diff
@@ -150,6 +150,9 @@
   and the codecov check itself would never execute.
   [(#5101)](https://github.com/PennyLaneAI/pennylane/pull/5101)
 
+* `qml.ctrl` called on operators with custom controlled versions will return instances of the custom class, and it will also flatten nested controlled operators to a single multi-controlled operation. For `PauliX`, `CNOT`, `Toffoli`, and `MultiControlledX`, calling `qml.ctrl` will always resolve to the best option in `CNOT`, `Toffoli`, or `MultiControlledX` depending on the number of control wires and control values.
```

Comment:
```suggestion
* `qml.ctrl` called on operators with custom controlled versions will return instances
  of the custom class, and it will also flatten nested controlled operators to a single
  multi-controlled operation. For `PauliX`, `CNOT`, `Toffoli`, and `MultiControlledX`,
  calling `qml.ctrl` will always resolve to the best option in `CNOT`, `Toffoli`, or
  `MultiControlledX` depending on the number of control wires and control values.
```